tests/pyconverter-test/cases/destructuring.py

```diff
@@ -0,0 +1,9 @@
+def initValue():
```

Comment:
Am I correct in understanding this will be picked up automatically as a test case simply because of its file path, then compared with the baseline by the same name?

Reply:
yup!